doc/Installation/Install-LibreNMS.md

```diff
@@ -0,0 +1,571 @@
+        
+        ```nginx
+        server {
+         listen      80;
+         server_name librenms.example.com;
+         root        /opt/librenms/html;
+         index       index.php;
+        
+         charset utf-8;
+         gzip on;
+         gzip_types text/css application/javascript text/javascript application/x-javascript image/svg+xml text/plain text/xsd text/xsl text/xml image/x-icon;
+         location / {
+          try_files $uri $uri/ /index.php?$query_string;
+         }
+         location /api/v0 {
+          try_files $uri $uri/ /api_v0.php?$query_string;
+         }
+         location ~ \.php {
+          include fastcgi.conf;
+          fastcgi_split_path_info ^(.+\.php)(/.+)$;
+          fastcgi_pass unix:/run/php-fpm-librenms.sock;
+         }
+         location ~ /\.ht {
+          deny all;
+         }
+        }
+        ```
+        
+        > NOTE: If this is the only site you are hosting on this server (it
+        > should be :)) then you will need to disable the default site.
+        
+        Delete the `server` section from `/etc/nginx/nginx.conf`
+        
+        ```
+        systemctl enable --now nginx
+        systemctl enable --now php-fpm
+        ```
+
+    === "Apache"
+        Create the librenms.conf:
+        
+        ```
+        vi /etc/httpd/conf.d/librenms.conf
+        ```
+        
+        Add the following config, edit `ServerName` as required:
+        
+        ```apache
+        <VirtualHost *:80>
+          DocumentRoot /opt/librenms/html/
+          ServerName  librenms.example.com
+        
+          AllowEncodedSlashes NoDecode
+          <Directory "/opt/librenms/html/">
+            Require all granted
+            AllowOverride All
+            Options FollowSymLinks MultiViews
+          </Directory>
+
+          # Enable http authorization headers
+          <IfModule setenvif_module>
+            SetEnvIfNoCase ^Authorization$ "(.+)" HTTP_AUTHORIZATION=$1
+          </IfModule>
+    
+          <FilesMatch ".+\.php$">
+            SetHandler "proxy:unix:/run/php-fpm-librenms.sock|fcgi://localhost"
+          </FilesMatch>
+        </VirtualHost>     
+        ```
+        
+        > NOTE: If this is the only site you are hosting on this server (it
+        > should be :)) then you will need to disable the default site. `rm -f /etc/httpd/conf.d/welcome.conf`
+        
+        ```
+        systemctl enable --now httpd
+        systemctl enable --now php-fpm
+        ```
+
+# SELinux
+
+=== "Ubuntu 20.04 / Debian 10"
+    SELinux not enabled by default
+
+=== "CentOS 8"
+    Install the policy tool for SELinux:
+    
+    ```
+    dnf install policycoreutils-python-utils
+    ```
+    
+    ## Configure the contexts needed by LibreNMS:
+    
+    ```
+    semanage fcontext -a -t httpd_sys_content_t '/opt/librenms/html(/.*)?'
+    semanage fcontext -a -t httpd_sys_rw_content_t '/opt/librenms/(logs|rrd|storage)(/.*)?'
+    restorecon -RFvv /opt/librenms
```

Comment:
CentOS 7 install also have `/opt/librenms/bootstrap/cache`. Is it not needed anymore to be allowed in selinux?

Reply:
the web server doesn't write to bootstrap/cache unless we run composer from it... which I don't think we will be doing.

Reply:
I'll drop it from the selinux :)